### DISCUSSION TITLE
Jingles / Station ID's Playlist Not Working

### DISCUSSION THREAD

**Discussion by BornKillaz** [AzuraCast/AzuraCast#7042] [Q&A] [answered]
Hi.

So, I have a default playlist, that runs with a priority of 2, with no schedule. 

Then, I have another playlist with the station ID's  and Jingles scheduled to play from 5am to midnight set to play shuffle once every 4 songs (set to play only one track at the scheduld time).

The problem is that the Jingles playlist never gets played.

Am I doing something wrong here?

Thanks in advance.

**Comment by gAlleb** [comment]
If you play jingles `Only play one track at scheduled time.` - It will only play `1` jingle from 5am to midnight and that's it! Uncheck that box.

**Reply by BornKillaz** [reply]
Hi. I appreciate your answer... it didn't work as you said, but pointed me in some kind of direction...

So, to get something out of that Jingles playlist, I had to put a check on:

- Interrupt other songs to play at scheduled time.
- Only play one track at scheduled time.

Also removed the scheduled option. 

With these options selected I sometimes, and this is very weird, sometimes I get a jingle to play, on top of a currently playing song (currently playing song crossfades to the jingle, the jingle plays and then the music continues)... But not always, sometimes it plays on top of the third song. Sometimes on the fourth, and most of the times the jingle does not play at all... I've set that Jingles playlist to play every 2 songs. 

I don't know what else to try. Nothing seems to work... It's a docker install on Debian 12. Azuracast on stable build.

**Reply by gAlleb** [reply, marked as answer]
Once again. If you wish for your jingles to play `Once per 4 Songs` in the Interval from `5AM to 12AM` - all you have to do is

1. Select play `Once per x Songs` and `4` in the `Number Section`
2. Select `Shuffled` Playback order.
3. Select `Hide Metadata from Listeners ("Jingle Mode")`
4. Select time interval `05:00 AM to 11:59 PM`

That's it. You don't need to use any advanced options:

>  Interrupt other songs to play at scheduled time.
    Only play one track at scheduled time.

Or any of other Advanced options. Because

1. `Interrupt other songs to play at scheduled time` - You simply don't need that. Because rotation of jingles won't work this way.
2. `Only play one track at scheduled time` - This function would play jingle only once within time interval `5AM to 12AM`. Only once. You want to play your jingles every 4 songs and yet you pick "another" option to play it `once`. They don't work together.

Please see the config for you to follow: 

1. 

<img width="401" alt="Снимок экрана 2024-04-04 в 19 56 05" src="https://github.com/AzuraCast/AzuraCast/assets/144447266/d2aaaccb-77ff-40c5-9b3e-401d4c5d7382">

2. 
<img width="401" alt="Снимок экрана 2024-04-04 в 19 56 50" src="https://github.com/AzuraCast/AzuraCast/assets/144447266/ada6d9a3-ac8c-4430-b758-aa1dbad82be3">

3. 
<img width="401" alt="Снимок экрана 2024-04-04 в 19 57 08" src="https://github.com/AzuraCast/AzuraCast/assets/144447266/1b87d4c4-7a3c-408f-bc3f-e83ba27b3f79">

**Reply by BornKillaz** [reply]
Ok, I must confess, I'm almost positive that I've tried already those settings... But I've set everything again as you've pointed out and it's WORKING! 

Thank you very much!

**Comment by Heimatradio** [comment]
After changes in the playlists, especially with scheduled times, you have to restart the AutoDJ

**Reply by gAlleb** [reply]
@Paulas-Welle Hi. No, you don't. Azuracast's `Nextsong API` will handle those changes. 
Unless you are in `Manual mode`.
